Ym2149Synth/SynthPatchStorage.h

```diff
@@ -20,6 +20,18 @@
  *
  */
 
+/*
+ * Check for Teensy LC
+ *
+ * LC has only 128 bytes of flash-emulated EEPROM
+ * so we lower the patch amount to 11 instead of 16
+ */
+#if defined (__MKL26Z64__)
+#define MAX_PATCHES 11
+#else
+#define MAX_PATCHES 16
+#endif
+
```

Comment:
Yes, so we just determine the maximum amount of patches available.
In the future there should probably be a MIDI CC number assigned for returning the maximum number of patches a YM2149 Synth can store, so any editor can easily adapt its interface.

With a Teensy 3.2 you could actually store a maximum of 186 11 byte patches, but that's probably overkill.